### PR TITLE
Fixes missing sqlite3 module in python3

### PR DESCRIPTION
**Pull request by misl6**
Fixes missing `_sqlite3` module in python3.

Addresses issue #314

### CODE REVIEW

File: recipes/python3/ModulesSetup [hunk 1]
```diff
@@ -2,9 +2,11 @@
 # Static compilation instructions for all binary modules.
 #####################################################################
 
+*static*
```
Comment (tito) [inline recipes/python3/ModulesSetup]: What is that?

File: recipes/python3/ModulesSetup [hunk 2]
```diff
@@ -2,9 +2,11 @@
...
 _asyncio _asynciomodule.c
 _bisect _bisectmodule.c
 _blake2 _blake2/blake2module.c _blake2/blake2b_impl.c _blake2/blake2s_impl.c
+_sqlite3 -I$(srcdir)/Modules/_sqlite -DMODULE_NAME='\"sqlite3\"' _sqlite/cache.c _sqlite/connection.c _sqlite/cursor.c _sqlite/microprotocols.c _sqlite/module.c _sqlite/prepare_protocol.c _sqlite/row.c _sqlite/statement.c _sqlite/util.c
```
Comment (tito) [inline recipes/python3/ModulesSetup]: Why the custom line doesn't include -DSQLITE_OMIT_LOAD_EXTENSION?

File: recipes/python3/__init__.py [hunk 3]
```diff
@@ -32,12 +32,39 @@ def prebuild_arch(self, arch):
         self.append_file("ModulesSetup.mobile", "Modules/Setup.local")
         self.apply_patch("xcompile.patch")
         self.set_marker("patched")
+    
+    def postbuild_arch(self, arch):
```
Comment (tito) [inline recipes/python3/__init__.py]: No, this should not be needed. Why this and not all the others modules declares in Modules?
Something is wrong here. if _ssl, _struct, array, _socket... are working, _sqlite3 should work the same, as it is declared the same way. Could it be the inclusion of `*static*` ?

File: recipes/python3/ModulesSetup (hunk 1 above)
Comment (misl6) [reply recipes/python3/ModulesSetup]: From *python* `Setup.dist`:

```# Each line in this file describes one or more optional modules.
# Modules configured here will not be compiled by the setup.py script,
# so the file can be used to override setup.py's behavior.
# Tag lines containing just the word "*static*", "*shared*" or "*disabled*"
# (without the quotes but with the stars) are used to tag the following module
# descriptions. Tag lines may alternate throughout this file.  Modules are
# built statically when they are preceded by a "*static*" tag line or when
# there is no tag line between the start of the file and the module
# description.  Modules are built as a shared library when they are preceded by
# a "*shared*" tag line.  Modules are not built at all, not by the Makefile,
# nor by the setup.py script, when they are preceded by a "*disabled*" tag
# line.
```

Basically, considering that is something by default, We can safely remove this line ( Already tried and it works, I just missed it over there during my tests)

File: recipes/python3/ModulesSetup (hunk 2 above)
Comment (misl6) [reply recipes/python3/ModulesSetup]: Is something by default, If We're more confident about having it in our `ModulesSetup` I can revert this file completely.

Comment (misl6) [reply recipes/python3/ModulesSetup]: I can confirm that reverting `ModulesSetup` to previous version leads to the same result.

File: recipes/python3/__init__.py (hunk 3 above)
Comment (misl6) [reply recipes/python3/__init__.py]: Not related to `*static*`. I also supposed that wasn't needed, but I've found out that for some reason, `python` itself builds this module as a `lib-dynload` also if it is supposed to "merge" it statically inside `libpython3`.

**Pull request merged**